src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1821,21 +1831,33 @@ components:
       type: object
       properties:
         raw_hex:
+          description: (false by default) If set to true, raw transaction hex is returned.
           type: boolean
         receipt_state_changes:
+          description: (false by default) if set to true, state changes inside receipt object are returned.
           type: boolean
         receipt_fee_summary:
+          description: (false by default) if set to true, fee summary inside receipt object is returned.
           type: boolean
         receipt_fee_source:
+          description: (false by default) if set to true, fee source inside receipt object is returned.
           type: boolean
         receipt_fee_destination:
+          description: (false by default) if set to true, fee destination inside receipt object is returned.
           type: boolean
         receipt_costing_parameters:
+          description: (false by default) if set to true, costing parameters inside receipt object is returned.
           type: boolean
         receipt_events:
+          description: (false by default) if set to true, events inside receipt object is returned.
+          type: boolean
+        receipt_output:
+          description: (true by default) If set to true, transaction receipt output is returned.
```

Comment:
The default value of `true` should be considered a bug over here, isn't it? This is a collection of opt-INs after all. It would be super confusing for everyone to have this single opt-in default to `true` for whatever reason. Then again fixing this bug would break backwards compatibility.

@dhedey What's our policy with those issues? Is it still a bug or is it a feature by now?

Reply:
agreed on slack that unfrotunately that's the best option to do it this way.